.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Before requesting gatekeeper:
+
+* [ ] Two review approvals and any prior change requests have been resolved.
+* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it
+      should be force landed and forced-landing tag is set
+
+### Gatekeeper:
+
+* [ ] You are the appropriate gatekeeper to be landing the patch.
+* [ ] The PR has 2 reviews by people familiar with the code.
+* [ ] Any appropriate watchers have had a chance to review the PR.
+* [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested
+      changes.
+* [ ] Githooks were used. If not, there is a sufficient reason to move forward.  Check copyrights
+      if githooks were not used.
+* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future
+      PRs.
+* [ ] All builds have passed.  Check non-required builds to ensure they don't indicate any problem
+      such as a compiler warning on extraneous platforms.
+* [ ] No new NLT or valgrind warnings.  Check the classic view. This step should only matter if
+      the build requires force landing.
+* [ ] Ensure sufficent testing is done.   Check feature pragmas and test tags.
```

Comment:
The gatekeeper should also verify that any tests skipped by the ticket sourcing the PR now pass with the PR changes